### DISCUSSION TITLE
Feature wish: variable number of windows in master for dynamic groups

### DISCUSSION THREAD

**Discussion by engblom** [stumpwm/stumpwm#1115] [Ideas]
When using other dynamic tiling wm, like xmonad, one can adjust the number of windows in the master area. Now that stumpwm got dynamic groups, it is possible to configure stumpwm to be similar to xmonad, but this feature seems to be missing.

**Comment by equwal** [comment]
I'm a really big fan of dwm's half-screen master area and half-screen compressed area thing. I don't know why you'd want more than one thing in the master area though when key-based resizing the master area is a better option.

**Comment by engblom** [comment]
> I'm a really big fan of dwm's half-screen master area and half-screen compressed area thing. I don't know why you'd want more than one thing in the master area though when key-based resizing the master area is a better option.

This feature allows to have 4 terminals arranged in a 2x2 grid. Or if one is monitoring continuously  5 terminals, one can have 2 in master and 3 in the stack, and all are kind of readable.  With xmonad I would press Mod+, to increase the number of windows in master, and once I do not need as many windows in master, I press Mod+. to decrease.
